frontend/src/hooks/useLookupCodes.tsx

```diff
@@ -46,8 +55,34 @@ function useCodeLookups() {
   };
 
   const getOptionsByType = (type: string) => getByType(type).map(mapLookupCode);
+
+  /**
+   * Return an array of SelectOptions containing property classifications.
+   * @function
+   * @param filter - A filter to determine which classifications will be returned.
+   * @returns
+   */
+  const getPropertyClassificationOptions = (
```

Comment:
Here's the real change.  We now enforce user based claims and can override.

Reply:
Does the @function provide value?

Reply:
Same with @returns if it is left blank.

Reply:
I've updated.